test/integration/etcd/BUILD

```diff
@@ -19,26 +19,23 @@ go_test(
     deps = [
```

Comment:
Can we please add a `test/integration/etcd/OWNERS` file?

Reply:
> Can we please add a test/integration/etcd/OWNERS file?

I took a shot at it.  It's a little weird because most of the time this test is touched as a result of adding to the API, so apireview is appropriate and most of them already have access.  The mechanics are important too, but that's almost entirely redhatters.

Luckily I think we can trust the top level test approvers to be responsible.